### ISSUE TITLE
Stealth disposals chute

### ISSUE THREAD

**Issue by fizzle7** [yogstation13/yogstation#2312]
You can hide a disposals chute under a closed airlock, and when you bump into the airlock to open it, you first go into the delivery chute which is completely retarded and is not how it should work.

**Comment by Slinky5890** [comment]
Damn, it was fun while it lasted

**Comment by fizzle7** [comment]
Not sure if I should reeeee at the "features" or abuse it in the future.

**Comment by Time-Green** [comment]
Feature. Don't fix

**Comment by fizzle7** [comment]
Yep, perfectly normal to quantum tunnel through the airlock into the chute. /s

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
why do you keep reporting all the fun features

**Comment by fizzle7** [comment]
again, "features", this makes no sense

**Comment by fonedsky** [comment]
because they're fucking fun you nerd

**Comment by fizzle7** [comment]
Lmao, do I look like I give a shit? It's not fun for everyone and it's a retarded bug that's considered a feature by shitlers.
